homeassistant/components/wink.py

```diff
@@ -41,13 +56,39 @@ def setup(hass, config):
     return True
 
 
-class WinkToggleDevice(ToggleEntity):
-    """Represents a Wink toggle (switch) device."""
+class WinkDevice(Entity):
+    """Represents a base Wink device."""
 
     def __init__(self, wink):
         """Initialize the Wink device."""
+        from pubnub import Pubnub
         self.wink = wink
         self._battery = self.wink.battery_level
+        if (SUBSCRIPTION_HANDLER is not None and
+                self.wink.pubnub_channel is not None):
+            self.sub_key = self.wink.pubnub_key
```

Comment:
Why are you storing these values on the object? Wouldn't this always be able to be retrieved using `self.wink.pubnub_key` ?